packages/runtime/client/rest/src/contract/transaction/endpoints/collection.ts

```diff
@@ -129,6 +133,19 @@ export const TransactionTextEnvelope = t.type({
   tx: TextEnvelopeGuard,
 });
 
+export type ApplyInputsToContractRequest = {
+  contractId: ContractId;
+  changeAddress: AddressBech32;
+  usedAddresses?: AddressBech32[];
+  collateralUTxOs?: TxOutRef[];
+  invalidBefore?: ISO8601;
+  invalidHereafter?: ISO8601;
+  version?: MarloweVersion;
```

Comment:
something to discuss for later, do we want to add version in the API or is this something fixed for the frontend that knows that it is compatible with certain version? 🤔